erlang/content.md

```diff
@@ -0,0 +1,28 @@
+# What is Erlang?
+
+Erlang is a programming language used to build massively scalable soft real-time systems with requirements on high availability. Some of its uses are in telecoms, banking, e-commerce, computer telephony and instant messaging. Erlang's runtime system has built-in support for concurrency, distribution and fault tolerance.
+
+> [wikipedia.org/wiki/Erlang_(programming_language)](https://en.wikipedia.org/wiki/Erlang_%28programming_language%29)
+
+%%LOGO%%
+
+# How to use this image
+
+## Create a `Dockerfile` in your Erlang app project
+
+```dockerfile
+FROM erlang:18-onbuild
```

Comment:
I'll remove these onbuild lines for now, will add the package manager `rebar` for the ONBUILD lines, it's like the pip for python
https://github.com/docker-library/python/blob/master/3.5/onbuild/Dockerfile